docs/pages/versions/unversioned/sdk/maps.mdx

```diff
@@ -0,0 +1,116 @@
+---
+title: Maps
+sidebar_title: Maps (expo-maps)
+description: A library that provides access to Google Maps on Android and Apple Maps on iOS.
+sourceCodeUrl: 'https://github.com/expo/expo/tree/main/packages/expo-maps'
+packageName: 'expo-maps'
+platforms: ['ios', 'android']
+isNew: true
+---
+
+import APISection from '~/components/plugins/APISection';
+import { APIInstallSection } from '~/components/plugins/InstallSection';
+import { Collapsible } from '~/ui/components/Collapsible';
+import { Step } from '~/ui/components/Step';
+import { Tabs, Tab } from '~/ui/components/Tabs';
+
+> **warning** **Expo Maps is currently in alpha and subject to breaking changes. It is not available in the Expo Go app.**
+
+## Installation
+
+<APIInstallSection />
+
+## Configuration
+
+Expo Maps provides access to the platform native map APIs on Android and iOS.
+
+- **Apple Maps is available on iOS only**. No additional configuration is required to use it after installing this package.
+- **Google Maps is available on Android only**. While Google provides a Google Maps SDK fore iOS, Expo Maps supports it exclusively on Android. If you want to use Google Maps on iOS, you can look into a using an [alternative library](https://reactnative.directory/) or [writing your own](/modules/overview/).
```

Comment:
```suggestion
- **Google Maps is available on Android only**. While Google provides a Google Maps SDK for iOS, Expo Maps supports it exclusively on Android. If you want to use Google Maps on iOS, you can look into using an [alternative library](https://reactnative.directory/) or [writing your own](/modules/overview/).
- **Apple Maps is available on iOS only**. No additional configuration is required to use it after installing this package.
```

- Fix platform order as per the writing style guide: https://github.com/expo/expo/blob/main/guides/Expo%20Documentation%20Writing%20Style%20Guide.md#platform-order-when-referencing-android-ios-and-web-or-multiple-platforms
- Fix typos

Reply:
i intentionally put ios first here because there is an explanation about configuration for google maps for android directly underneath, so i wanted to put those two together